install.py

```
@@ -452,3 +454,36 @@ def main():
+    pulid_dir = "PuLID"
+    if not os.path.exists(pulid_dir):
+        print(f"⚠️ {pulid_dir} directory not found, skipping PuLID setup")
+        return
+
+    original_dir = os.getcwd()
+    try:
+        os.chdir(pulid_dir)
+        pulid_env_path = ".venv"
+
+        if not os.path.exists(pulid_env_path):
+            print(f"🔧 Creating PuLID isolated virtual environment...")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)